app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,20 @@
+<div id="<%= dom_id group %>">
+
+<div class="group-display">
+
+<div class="group-list">
+
+  <div class="img"><img src="<%= group.icon %>"> </div>
```

Comment:
### OPTIONAL

- [ ] The  icon image is currently broken kindly make sure  images are  not broken so users can see images uploaded
- [ ]  Kindly add alt texts to all image tag, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible